widgets/common/src/filters/inputs/MultipleStringValuesInput.tsx

```diff
@@ -0,0 +1,35 @@
+import { FunctionComponent, useMemo } from 'react';
+
+interface MultipleStringValuesInputProps {
+    name: string;
+    onChange: (newValue: string[]) => void;
+    placeholder: string;
+    value: string[];
+}
+
+const MultipleStringValuesInput: FunctionComponent<MultipleStringValuesInputProps> = ({
+    name,
+    onChange,
+    placeholder,
+    value
+}) => {
+    const { Dropdown } = Stage.Basic;
+    const options = useMemo(() => value.map(element => ({ text: element, value: element })), [value]);
```

Comment:
Optional:

Is there a type we could use for the return type of the mapping function to get a better error message if it does not match the expected type? I think we could do

```suggestion
    const options = useMemo(() => value.map((element): DropdownItemProps => ({ text: element, value: element })), [value]);
```

Reply:
Sure, make sense. Applied.